new/LIT7337MiracleMarySycomore.xml

```diff
@@ -0,0 +1,81 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7337MiracleMarySycomore" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:id="t1">Miracle of Mary: Miracle of the flight of Egypt; Miracle of the sycomore of Dabra Mǝṭmāq</title>
```

Comment:
sycamore

Reply:
(I have added the link to the issue in the description, please always remember to link pull requests to issues they respond to, thank you)

Reply:
ok, thanks!